src/main/java/io/datanerds/avropatch/operation/Replace.java

```diff
@@ -12,20 +12,20 @@
     @AvroIgnore
     public static final String op = "replace";
```

Comment:
I will remove the literals completely for now. I've added them initially to comply with the JSON PATCH format. Since we're not enforcing on it we can add the literals again as soon we consider supporting JSON serialization.